www/footer.php

```diff
@@ -138,6 +138,20 @@
 	</div>
 </div>
 
+<!-- Lyrics -->
```

Comment:
Maybe a little bit overkill to add it also to the main menu ? I think on the play context menu would be enough.